### PR TITLE
Set up a minimal CI

### PR DESCRIPTION
**Pull request by lebaudantoine**
## Purpose

Maintain high quality code standards + Improve DX while reviewing code

## Proposal

Set up a minimal CI configuration which will lint git commits, code sources, and build static sources.
Also, we will maintain dependencies up-to-date using renovate bot.

As soon as the PR is merged, I will set up these two jobs as required to merge any PR.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/Bug_report.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+**Expected behavior/code**
+A clear and concise description of what you expected to happen (or code).
+
+**Steps to Reproduce**
+1. Do this...
+2. Then this...
+3. And then the bug happens!
+
+**Environment**
+- People version:
```
Comment (manuhabitela) [inline .github/ISSUE_TEMPLATE/Bug_report.md:21]: I don't think I'm a machine, I don't have any version number that I now of

File: .github/ISSUE_TEMPLATE/Feature_request.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+A clear and concise description of what the problem is. For example: I need to do some task and I have an issue...
+
+**Describe the solution you'd like**
+A clear and concise description of what you want to happen. Add any considered drawbacks.
+
+**Describe alternatives you've considered**
+A clear and concise description of any alternative solutions or features you've considered.
+
+**Discovery, Documentation, Adoption, Migration Strategy**
+If you can, explain how users will be able to use this and possibly write out a version the docs (if applicable).
```
Comment (manuhabitela) [inline .github/ISSUE_TEMPLATE/Feature_request.md:19]:
```suggestion
If you can, explain how users will be able to use this and possibly write out a version in the docs (if applicable).
```

i guess

File: .github/ISSUE_TEMPLATE/Support_question.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+about: If you have a question 💬, or something was not clear from the docs!
+
+---
+
+<!-- ^ Click "Preview" for a nicer view! ^
+We primarily use GitHub as an issue tracker. If however you're encountering an issue not covered in the docs, we may be able to help! -->
+
+---
+
+Please make sure you have read our [main Readme](https://github.com/numerique-gouv/people).
```
Comment (manuhabitela) [inline .github/ISSUE_TEMPLATE/Support_question.md:12]: wrong link :)

File: .github/ISSUE_TEMPLATE/Support_question.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+Also make sure it was not already answered in [an open or close issue](https://github.com/numerique-gouv/people/issues).
```
Comment (manuhabitela) [inline .github/ISSUE_TEMPLATE/Support_question.md:14]: wrong link :)

Comment (manuhabitela) [inline .github/ISSUE_TEMPLATE/Support_question.md:14]:
```suggestion
Also make sure it was not already answered in [an open or closed issue](https://github.com/numerique-gouv/people/issues).
```

File: .github/ISSUE_TEMPLATE/Support_question.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+If your question was not covered, and you feel like it should be, fire away! We'd love to improve our docs! 👌
+
+**Topic**
+What's the general area of your question: for example, docker setup, database schema, search functionality,...
```
Comment (manuhabitela) [inline .github/ISSUE_TEMPLATE/Support_question.md:19]:
```suggestion
What's the general area of your question: for example, docker setup, database schema, search functionality…
```

you are not the _king_ of nitpicking my bro

File: .github/workflows/lasuite.yaml [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+    branches:
+      - '*'
+
+jobs:
+  lint-git:
+    runs-on: ubuntu-latest
+    if: github.event_name == 'pull_request' # Makes sense only for pull requests
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
```
Comment (manuhabitela) [inline .github/workflows/lasuite.yaml:19]: i'm really not a user of github actions, just asking here: any reason to specificly use v2 ?

**Review by manuhabitela — APPROVED** [review]
Didn't really test it as it's kinda annoying to do for those CI things.

But seems great to me looking at it, great job :)

File: .github/workflows/lasuite.yaml (hunk 6 above)
Comment (lebaudantoine) [reply .github/workflows/lasuite.yaml:19]: No, just being aligned with other project CI, and I am not even sure that's a good reason. People CI might be outdated too. As it's not a big deal, I'll just ignore this for now and discuss it with @rouja !

**Review by manuhabitela — APPROVED** [review]

**Pull request merged**